content/research/presentations.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Presentations"
+date: 2021-05-28T09:44:45-05:00
+draft: false
+layout: page
+---
+
+
+{{< presentation preview_type="pdf" info_text="**Yssavo Camacho-Neves** presented a poster on “500 Days in the Photospheric Phase of Type Iax Supernova 2014dt” at *SuperVirtual 2021* in which she used TARDIS for modelling the spectrum of SN 2014dt." embed_link="https://drive.google.com/file/d/1GDHhw8q8lKhErIA-tMSRb7Mun6DdNoVK/preview">}}
```

Comment:
This is excellent - exactly what I wanted.

Not sure why you have remove Posters and Talks subheadings and brief explanation (right after presentations title) we had earlier - please include them back as they were. You can mix and match shortcodes and markdown.

Reply:
Added them back in e6107cf